custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -124,6 +124,9 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         {id: 8, name: 'AWW Performance Report'},
         {id: 9, name: 'LS Performance Report'},
     ];
+    if (vm.haveAccessToFeatures) {
```

Comment:
have you tested this that it does not show when feature flag is off?

Reply:
Hi, it was manually tested, I didn't have to write another test, one checking this behavior was already written, it's in 'Download Directive dont have access to features', and looks like this:

it('tests that all users have access to ISSNIP monthly register', function () {
    var length = controller.indicators.length;
    assert.equal(9, length);
});

Regards,
Maciej

Reply:
Yeah, I am talking about manual tests. If it's tested manually then it's fine.